tests/Unit/Domain/Creators/Python/Test_Cylinder.py

```diff
@@ -0,0 +1,22 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre.Domain.Creators import Cylinder, DomainCreator3D
+import unittest
+
+
+class TestCylinder(unittest.TestCase):
+    def test_construction(self):
+        cylinder = Cylinder(inner_radius=1,
```

Comment:
We're not as strict on python style, but it might still be helpful to include the decimal points for the quantities that are floating-point (e.g `1.0` here)

Reply:
Added the decimal point. However, the PEP8 style doesn't give a guideline on this and I don't think we have to be strict about it in Python. In Python ["Readability counts"](https://www.python.org/dev/peps/pep-0020/) and it's probably more readable to omit unnecessary characters